src/components/utilities/dialog.tsx

```diff
@@ -99,18 +99,28 @@ export class DialogComponent extends BaseComponent<IProps, IState> {
   }
 
   private renderPromptContents(dialog: UIDialogModelType) {
+    const input = dialog.rows
```

Comment:
Should this be `dialog.rows > 1`, i.e. should explicitly specifying `rows: 1` be considered a request for multi-line or single-line input?

Reply:
Good point.  I'll leave it as is for now.